temoa/cli.py

```
@@ -437,6 +439,165 @@ def migrate(
+        if 'input_database' in doc:
+            doc['input_database'] = f'{new_database_name}.sqlite'
+
+        if 'output_database' in doc:
+            doc['output_database'] = f'{new_database_name}.sqlite'
+
+        # Write back with tomlkit
+        with open(config_path, 'w', encoding='utf-8') as f:
+            tomlkit.dump(doc, f)
+
+    except Exception as e:
+        logger.warning(f'Failed to update TOML configuration {config_path}: {e}')
```

Ruff (0.14.4)

Logging statement uses f-string

(G004)